docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -277,38 +282,93 @@ Scale-up of breast cancer screening coverage among insured population
+
+   1) Sex
+   2) Age 
+   3) Family history
+   4) With diagnosis of DCIS/LCIS 
+
+  .. image:: breast_cancer_screening_tree_China2.svg
+
+
++--------------------------------------------------------------------------------------------------+
+| Screening branches                                                                               | 
++========+========+=======+==========+===========+=====================+=============+=============+
+| Branch | Sex    | Age   | Family   | With DCIS | Screening           | Sensitivity | Specificity |
+|        |        | group | history  | or LCIS   | tech                |             |             |
++--------+--------+-------+----------+-----------+---------------------+-------------+-------------+
+| A      | Female | 30-69 | Yes      | either    | MRI, every year     | 91%         | 100%        |
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+                                             
+| B      |        | 30-44 | No       | Yes       | BUS, every year     | 73.7%       | 100%        |      
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+   
+| C      |        | 45-69 | No       | Yes       | MAM+BUS, every year | 93.9%       | 100%        |      
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+    
+| D      |        | 30-69 | No       | No        | MAM, every 2 years  | 84.8%       | 100%        |
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+    
+| E      |        | <30   | either   | either    | No screening                                    |
+|        |        | or 70+|          |           |                                                 |
++--------+--------+-------+----------+-----------+-------------------------------------------------+
+| F      | Male   | any   | either   | either    | No screening                                    |
++--------+--------+-------+----------+-----------+-------------------------------------------------+
+| MAM: mammography; BUS: breast ultrasound                                                         |
++--------------------------------------------------------------------------------------------------+
+ 
+ 
+.. todo::
+  The breast cancer recommendation says: "Women aged 69 years and older and with an average risk of breast cancer should have the opportunity to continue screening as long as their overall health is good and they have a life expectancy of 10 years or longer." I'm wondering if this only applies to average risk women and that high risk 70+ women would continue to have MRI (follow branch A). Yongquan, do you see any recommendation that separates out average risk and high risk 70+ women?   
+
```

Comment:
Let's not assign any screening practice for women 70+ in China as our client excluded this age from eligible group for regular breast cancer screening.